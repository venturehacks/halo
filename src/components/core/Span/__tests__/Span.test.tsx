import { render } from 'enzyme';
import React from 'react';

import { Span } from '../index';

test('Smoke test', () => {
  const span = render(<Span>Bueno</Span>);
  expect(span.text()).toEqual('Bueno');
});
