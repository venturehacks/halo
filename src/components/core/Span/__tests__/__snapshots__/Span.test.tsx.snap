// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Span snapshots contrast 1`] = `
<span
  class="__halo_textContrast_dark_AA"
>
  AA
</span>
`;

exports[`Span snapshots muted 1`] = `
<span
  class="__halo_textContrast_dark_AAA"
>
  AAA
</span>
`;

exports[`Span snapshots xmuted 1`] = `
<span
  class="__halo_textContrast_dark_AA"
>
  AA
</span>
`;

exports[`Span snapshots xxmuted 1`] = `
<span
  class="__halo_textContrast_dark_A"
>
  A
</span>
`;
